homeassistant/components/tuya/switch.py

```diff
@@ -88,6 +88,20 @@
             name="Switch",
         ),
     ),
+    # Irrigator
+    # https://developer.tuya.com/en/docs/iot/categoryggq?id=Kaiuz1qib7z0k
+    "ggq": (
+        SwitchEntityDescription(
+            key=DPCode.CHILD_LOCK,
```

Comment:
I don't see the child lock in the specifications, nor in your provided data dump?

Reply:
Correct, removed.